.github/labeler.yml

```diff
@@ -1,20 +0,0 @@
-# Union of api, connectors, documentation, octavia-cli, CDK and normalization.
-area/api:
-  - airbyte-api/*
-  - airbyte-api/**/*
-
-area/connectors:
-  - airbyte-integrations/connectors/*
-  - airbyte-integrations/connectors/**/*
-
-area/documentation:
-  - docs/*
-  - docs/**/*
-
-CDK:
-  - airbyte-cdk/*
-  - airbyte-cdk/**/*
-
-normalization:
-  - airbyte-integrations/bases/base-normalization/*
-  - airbyte-integrations/bases/base-normalization/**/*
```

Comment:
I think we should keep `area/documention`, `area/connectors` and `CDK` (for the java cdk).

Reply:
These are not in fact used in any workflows, and I have not seen us use _pr labels_ that are not specific connectors in any of our data pipelines.